organization/strategic-council.md

```diff
@@ -20,16 +20,24 @@ Members of the strategic council are expected to:
 * follow along with briefings sent by the board of directors
 * participate in the council's (email) discussions
 
-This commitment is estimated to be an average of 15 minutes per week.
 This is a voluntary role and non-remunerated.
-Membership of the strategic council can be terminated at any time in writing by either the member of the strategic council or the board of directors.
 
 ## Appointments
 
 These are the steps for appointing a new strategic council member.
 
-1. Potential new strategic council members are either invited to apply by the Foundation for Public Code, or may nominate themselves.
+1. Potential new strategic council members are either invited to apply by the Foundation for Public Code, or may nosminate themselves.
```

Comment:
I believe that nominate is now misspelled. It does not contain an S. Apologies if I am a poor reader of diff files.

Reply:
I believe this is now fixed! Thanks for checking.